app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> This Project</h1>
+
+<h2>Project name: <%= @project.name %></h2>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
```

Comment:
Ahhh, yes, so you did include a label here, so in your test, it's more explicit if you check for `Challenge Theme: Recycle Materials`. (Refer to my comment in the spec file.